Package.swift

```diff
@@ -10,8 +10,9 @@ let package = Package(
       dependencies: [.Target(name: "SourceKittenFramework")]),
   ],
   dependencies: [
-    .Package(url: "https://github.com/drmohundro/SWXMLHash.git", majorVersion: 2),
-    .Package(url: "https://github.com/Carthage/Commandant.git", majorVersion: 0, minor: 8),
-    .Package(url: "https://github.com/norio-nomura/swift-corelibs-xctest.git", majorVersion: 0),
+    .Package(url: "https://github.com/jpsim/SourceKit", majorVersion: 1),
+    .Package(url: "https://github.com/drmohundro/SWXMLHash", majorVersion: 2),
+    .Package(url: "https://github.com/Carthage/Commandant", majorVersion: 0, minor: 8),
+    .Package(url: "https://github.com/norio-nomura/swift-corelibs-xctest", majorVersion: 0),
```

Comment:
On `make spm_test` in SwiftLint:
> Cloning https://github.com/norio-nomura/swift-corelibs-xctest
> Using version 0.1.0-test of package swift-corelibs-xctest
> Cloning https://github.com/behrang/YamlSwift.git
> Using version 1.4.2 of package YamlSwift
> Cloning https://github.com/norio-nomura/swift-corelibs-xctest.git
> /usr/local/bin/git clone --recursive --depth 10 https://github.com/norio-nomura/swift-corelibs-xctest.git /Users/norio/Documents/workspace/github/SwiftLint/Packages/swift-corelibs-xctest
> fatal: destination path '/Users/norio/Documents/workspace/github/SwiftLint/Packages/swift-corelibs-xctest' already exists and is not an empty directory.
>
> error: Failed to clone https://github.com/norio-nomura/swift-corelibs-xctest.git to /Users/norio/Documents/workspace/github/SwiftLint/Packages/swift-corelibs-xctest

It seems we need making *exactly same urls* in each `Package.swift`, can not mix with/without `.git` on same repo. :confused: